client/components/LearnerViewDirective.js

```diff
@@ -477,10 +477,22 @@ tie.directive('learnerView', [function() {
             feedbackDiv.scrollTop = feedbackDiv.scrollHeight +
               additionalHeightForLoadingIndicator;
             $timeout(function() {
-              SolutionHandlerService.processSolutionAsync(
-                tasks[currentTaskIndex], code,
+              var index = 0;
```

Comment:
We proposed a design for solving this problem
https://docs.google.com/a/google.com/document/d/1HmmorRKmaV6IVqUJYDSBBEYRAzn-MK1y3HLctkOSTJ8/edit?usp=sharing
Please review that.